mlflow/models/evaluation/base.py

```diff
@@ -1289,6 +1289,8 @@ def evaluate(
           parameter will be ignored.
         - **sample_weights**: Weights for each sample to apply when computing model performance
           metrics.
+        - **col_mapping**: A dictionary mapping column names in the input dataset or output
+          predictions to column names used when invoking the evaluation functions.
```

Comment:
Can we add an example for this?

Reply:
I will add this in the examples section we are adding soon. In this section i think we just need to mention it in the list of evaluator configs.